### PR TITLE
feat: remove exp/term

### PR DESCRIPTION
**Pull request by caarlos0**
it was moved/refactored into `/term`, `/ansi`, `/input`...

### CODE REVIEW

File: go.work.sum [hunk 1]
```diff
@@ -7,6 +7,8 @@ github.com/charmbracelet/harmonica v0.2.0/go.mod h1:KSri/1RMQOZLbw7AHqgcBycp8pgJ
 github.com/charmbracelet/lipgloss v0.11.0/go.mod h1:1UdRTH9gYgpcdNN5oBtjbu/IzNKtzVtb7sqN1t9LNn8=
 github.com/charmbracelet/lipgloss v0.12.0/go.mod h1:beLlcmkF7MWA+5UrKKIRo/VJ21xGXr7YJ9miWfdMRIU=
 github.com/charmbracelet/ssh v0.0.0-20240515141028-546b2ee33a4d/go.mod h1:8/Ve8iGRRIGFM1kepYfRF2pEOF5Y3TEZYoJaA54228U=
+github.com/charmbracelet/x/errors v0.0.0-20240117030013-d31dba354651/go.mod h1:2P0UgXMEa6TsToMSuFqKFQR+fZTO9CNGUNokkPatT/0=
+github.com/charmbracelet/x/exp/term v0.0.0-20240524151031-ff83003bf67a/go.mod h1:YBotIGhfoWhHDlnUpJMkjebGV2pdGRCn1Y4/Nk/vVcU=
```
Comment (aymanbagabas) [inline go.work.sum:11]: Still referenced here :/

**Review by aymanbagabas — APPROVED** [review]

File: go.work.sum (hunk 1 above)
Comment (caarlos0) [reply go.work.sum:11]: yeah, tracking

Comment (caarlos0) [reply go.work.sum:11]: some dependency still using the old x/exp/term, I think `huh`

**Review by aymanbagabas — APPROVED** [review]

File: go.work.sum (hunk 1 above)
Comment (decentral1se) [reply go.work.sum:11]: > some dependency still using the old x/exp/term, I think `huh`

fyi i think i ran into that: https://github.com/charmbracelet/huh/issues/361

**Pull request merged**